src/coffee/import.coffee

```diff
@@ -43,10 +43,12 @@ class Import
     options.importFormat = options.importFormat || "csv"
     options.csvDelimiter = options.csvDelimiter || ","
     options.encoding = options.encoding || "utf-8"
+    options.mergeCategoryOrderHints = !!options.mergeCategoryOrderHints
```

Comment:
This will change all possible values into `Boolean`. If you provide an input parameter it will be `true` otherwise `false`.